app/models/rental_unit.rb

```diff
@@ -10,4 +10,15 @@ def price
   def price_per_room
     price / rooms
   end
+
+  def total_applicable_price
+    if price_per_room > 5000
```

Comment:
Consider adding an `else` clause to handle the case when `price_per_room` is less than or equal to 5000. Currently, the method `total_applicable_price` will return `nil` in this case, which may not be the expected behavior.